utils/wasm-instrument/Cargo.toml

```diff
@@ -17,14 +17,12 @@ enum-iterator.workspace = true
 [dev-dependencies]
 wasmparser.workspace = true
 wat.workspace = true
-gear-backend-wasmi.workspace = true
+gear-backend-wasmi = { workspace = true, features = ["std"] }
```

Comment:
We can remove `gear-backend-wasmi` now and not publish it.

Reply:
#3199 

`gcli` currently requires this crate for processing metadata xd